xdsl/ir.py

```diff
@@ -659,6 +659,30 @@ def add_region(self, region: Region) -> None:
         self.regions.append(region)
         region.parent = self
 
+    def get_region_index(self, region: Region) -> int:
+        """Get the region position in the operation."""
```

Comment:
```suggestion
        """Get the index of a region attached to this operation."""
```

Reply:
That would be `region_segment_sizes`, distinct from `operand_segment_sizes`
AFAIU, this is an xDSL exclusivity (:tada:) that @math-fehr thought about, but we're not sure if it actually makes sense, so we're thinking about keeping it or not. So, at the moment, I'm not sure we want to fully test and support it?

Reply:
Ah yeah, I guess I am confused, don't bother then.